pkg/workloads/cortex/internal/setup.py

```diff
@@ -0,0 +1,50 @@
+# Copyright 2020 Cortex Labs, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import pathlib
+import pkg_resources
+from setuptools import setup, find_packages
+
+with pathlib.Path("requirements.txt").open() as requirements_txt:
+    install_requires = [
+        str(requirement) for requirement in pkg_resources.parse_requirements(requirements_txt)
+    ]
+
+setup(
+    name="cortex-internal",
+    version="master",  # CORTEX_VERSION
+    description="Internal package for Cortex containers",
+    author="cortex.dev",
+    author_email="dev@cortex.dev",
+    license="Apache License 2.0",
+    url="https://www.cortex.dev",
```

Comment:
nit: consider using the repo link instead

Reply:
Yep, will do so.